Makefile

```
@@ -0,0 +1,18 @@
+S3CMD_VERSION	= $(shell cat VERSION)
+IMAGE_VERSION	= $(S3CMD_VERSION)-$(shell git describe --always --tags)
+
+default: Dockerfile
+	docker build -t infobloxcto/s3cmd:$(IMAGE_VERSION) \
```

```suggestion
	docker build -t infoblox/s3cmd:$(IMAGE_VERSION) \ 
```
The image will be opensourced or private ?

---

open source. Do the open source images go to infoblox/ instead of infobloxcto/?

---

I think so.
https://github.com/infobloxopen/atlas-gentool/blob/master/Makefile#L3

---

fixed